test/MusicStore.E2ETests/SmokeTestsUsingStore/TestHelper.cs

```diff
@@ -49,7 +52,7 @@ public TestHelper(ITestOutputHelper output) : base(output)
                         DbUtils.CreateConnectionString(musicStoreDbName)));
 
                 deploymentParameters.EnvironmentVariables.Add(
-                    new KeyValuePair<string, string>("DOTNET_SHARED_STORE", storeDirectory));
+                    new KeyValuePair<string, string>("DOTNET_SHARED_STORE", Environment.GetEnvironmentVariable("DOTNET_SHARED_STORE")));
```

Comment:
Don't child processes automatically inherit all environment variables? Or maybe it's too late at night to remember this detail correctly.